packages/ocamlformat_support/ocamlformat_support.0.2/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Pierre Weis"
+homepage: "https://github.com/ocaml-ppx/ocamlformat/tree/support"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+license: "LGPL-2 with OCaml linking exception"
+dev-repo: "https://github.com/ocaml-ppx/ocamlformat.git#support"
+build: [make]
```

Comment:
`format_.ml` is generated by `make`, which seemed to be sufficient, but I have observed in the past that the behavior of `{pin}` was not always what I expected.

But, the busybox patch is unusably feeble, so I'm sending an update with no patching and this issue will be moot.

Reply:
hum I think I am a bit confused because `format_.ml` in the archive doesn't contain `pp_print_break_or_string_if_newline` as well (which seems to be required to make `ocamlformat` compile).

Maybe just commit the patched file and "keep" the diff as a separated commit would be enough?

Reply:
https://github.com/ocaml/opam-repository/pull/11083 (which this PR is rebased on top of) adds a dependency to ocamlformat on ocamlformat_support 0.1 in order to avoid a compilation failure due to `pp_print_break_or_string_if_newline`. The intent is for ocamlformat_support 0.2 to enter the repo only after ocamlformat 0.2 depends on 0.1. The upcoming ocamlformat 0.3 will use the updated ocamlformat_support.